src/runtime_src/driver/xclng/drm/xocl/devices.h

```diff
@@ -962,5 +971,8 @@ enum {
 #define	XOCL_USER_QDMA_PCI_IDS						\
 	{ XOCL_PCI_DEVID(0x10EE, 0x6AA0, 0x4360, USER_QDMA) }
 
+#define XOCL_DSA_VBNV_MAP						\
+	{ 0x10EE, 0x5001, PCI_ANY_ID, "xilinx_u200_xdma_201820_1",	\
+		&XOCL_BOARD_USER_XDMA }
```

Comment:
Yes. you may create XDMA_CUSHIFT for u200_xdma_201820_1.
If the old DSA has unique vendor,device,subdevice combination, you may change PCI ID table as we did before. this vbnv_map is created to cover the case that DSAs which have same vendor,device,subdevice combination but different HW settings.
I am not sure I understand your last question. VBMV_MAP uses PCI_IDS and VBNV as the unique key , and it will overwrite the settings we have in PCI ID table.

Reply:
mm... wait. my code seems only take care subdevices but not the flags. I will think about this more